### ISSUE TITLE
path-style suggests strings for arrays with non-constant values

### ISSUE THREAD

**Issue by Wilfred** [wix/eslint-plugin-lodash#106]
The following code produces a warning with the default lodash configuration:

```js
function foo(x, y) {
  return _.get(x, ['bar', y, 'baz']); // Use a string for paths [lodash/path-style]
}
```

I don't think the warning should fire in this case. I could write `_.get(x, 'bar.' + y + '.baz')` (or use string interpolation in ES6) but I can't use a simple string literal in this case.

**Comment by mik01aj** [comment]
@Wilfred agree. This also has other semantics when `y` contains dots.
